master/internal/api_experiment.go

```diff
@@ -312,7 +323,7 @@ func (a *apiServer) KillExperiment(
 		return nil, err
 	}
 
-	addr := actor.Addr("experiments", req.Id).String()
```

Comment:
I think it's fine how it was, personally. Any reason for this?

Reply:
just remove/lower the possibility of typos when typing the address.